packages/runtime/runtime-definitions/src/summary.ts

```diff
@@ -102,6 +104,12 @@ export interface ISummarizerNode {
      * @param fullTree - true to skip optimizations and always generate the full tree
      */
     summarize(fullTree: boolean): Promise<ISummarizeResult>;
+    /**
+     * Checks if there are any additional path parts for children that need to
+     * be loaded from the base summary.
+     * @param snapshot - the base summary to parse
+     */
+    loadBaseSummaryWithoutDifferential(snapshot: ISnapshotTree): void;
```

Comment:
No I don't think so, it could be called in the DDS layer as well. It isn't something special about the root, just that the root is opting out of differential summaries by design vs. by configuration.

Reply:
And by opting out, we don't need to use the async version.

Reply:
Okay, so this is for differential summaries. I am not very familiar with me so its a little difficult for me to follow. Do we still need all this since differential summaries is disabled?

Reply:
It's not for differential summaries, it's specifically because we don't need differential summaries.

We need a way for the SummarizerNode to know the path to itself & its children for the summary which we loaded from. That's a main reason why we have `loadBaseSummary`. Before this change, differential summaries were the only reason the path could be different from just the ID, but now we have to route through .channels subtrees so we run that logic and store that path when we call `loadBaseSummary` or `loadBaseSummaryWithoutDifferential`.